### DISCUSSION TITLE
What's the difference between `ColorJitter` and `ColorJiggle`?

### DISCUSSION THREAD

**Discussion by kunaltyagi** [kornia/kornia#1849] [Questions] [answered]
The documentation is nearly the same (I do see a difference in the source code, but the functions used inside Jitter aren't documented as well)

**Comment by edgarriba** [comment, marked as answer]
`ColorJitter` tries to match torchvision implementation while `ColorJiggle`  follows strictly the theory behind image processing discussed in previous threads: https://github.com/kornia/kornia/issues/565

Kornia team will maintain only `ColorJiggle`

**Reply by kunaltyagi** [reply]
Great. Is it possible to add more details to the source code to make it simpler for people to choose between them?
* Explicitly state `ColorJitter` is not going to be maintained, and is solely for compatibility with torchvision (and indirectly PIL)
* `ColorJiggle` follows the theory of image processing more strictly and is the preferred method

**Reply by edgarriba** [reply]
Definitely and you are very welcomed to send a quick pr updating the docs

**Reply by kunaltyagi** [reply]
All docs are inline with the code, right?

**Reply by edgarriba** [reply]
Yes , just need to modify the docstring in code
